src/components/layout/Footer.tsx

```
@@ -1,21 +1,54 @@
+  if (shouldHide) return null;
+
+  const year = new Date().getFullYear();
+
   return (
-    <footer className="mt-14 flex w-full justify-center bg-gray-50 md:mt-20">
-      <div className="w-full max-w-6xl px-6 py-8 md:px-16 md:py-10">
-        <a
-          target="_blank"
+    <Flex
+      as="footer"
+      role="contentinfo"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<footer>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)